scania-digital-design-system/sdds
# fix(checkbox): moving theming rules to component folder

**Describe pull-request**  
Moving theming rules to component folder

**Solving issue**  
Fixes: [AB#1738](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1738)

**How to test**  
1. Open Checkbox in Storybook of this branch
2. Open Checkbox in Storybook of production
3. Compare colors - there should not be any difference between those two
